docs/self-managed/platform-deployment/overview.md

```diff
@@ -47,7 +47,7 @@ For production usage in Camunda 8 Self-Managed, we highly recommend using a real
 We support the following deployment options (the sequence expresses preference) for production:
 
 1. [**SaaS**](https://camunda.com/get-started)
-2. [**Helm/Kubernetes**](./helm-kubernetes/overview.md) independent of where this is hosted, for example OpenShift, EKS, or GKE.
```

Comment:
I'd call it Helm since K8s has many other methods anyway (also because Helm works on every K8s distro like OpenShift).

Reply:
Will any deployment on K8s be something we support (from a supported environments perspective)? We need to be as clear as possible.